com.reprezen.swagedit/src/com/reprezen/swagedit/messages.properties

```diff
@@ -22,7 +22,7 @@ error_additional_properties_not_allowed = object has properties %s which are not
 error_required_properties = object has missing required properties %s
 error_duplicate_keys = Duplicate key: the object has more than one property named %s
 error_cannot_read_content = Unable to read content. It may be invalid YAML
-error_missing_reference= Invalid Reference - SwagEdit was unable to resolve the reference. \n\
+error_missing_reference= Invalid Reference - KaiZen Editor was unable to resolve the reference. \n\
```

Comment:
This i18n binding is not used in the editor code, I only found usages in `ReferenceValidatorTest`. We can remove it to avoid code duplication and replace messages in tests with the messages defined in the core plugin.